tuxemon/event/actions/menu.py

```diff
@@ -0,0 +1,62 @@
+# SPDX-License-Identifier: GPL-3.0
+# Copyright (c) 2014-2023 William Edwards <shadowapex@gmail.com>, Benjamin Bean <superman2k5@gmail.com>
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Optional, final
+
+from tuxemon.event.eventaction import EventAction
+
+
+@final
+@dataclass
+class MenuAction(EventAction):
+    """
+    Enable/disable one or more menu.
+
+    Script usage:
+        .. code-block::
+
+            disable_menu [menu]
+
+    Script parameters:
+        act: enable or disable
+        menu: specific menu (menu_monster, menu_bag, menu_player,
+            exit, menu_options, menu_save, menu_load)
+            without specification, everything disabled
+
+    eg. menu disable,menu_bag
+    eg. menu enable,menu_player
+
+    """
+
+    name = "menu"
+    act: str
+    menu: Optional[str] = None
+
+    def start(self) -> None:
+        player = self.session.player
+        result = True if self.act == "enable" else False
+
+        if self.menu is None:
+            player.menu_bag = result
+            player.menu_load = result
+            player.menu_monsters = result
+            player.menu_save = result
+            player.menu_options = result
+            player.menu_player = result
+
+        if self.menu and self.menu == "menu_bag":
```

Comment:
i think a switch statement would fit better here.

Reply:
you're right, it looks bad